packages/astro/src/cli/index.ts

```diff
@@ -109,43 +111,51 @@ export async function cli(args: string[]) {
 	} else if (flags.silent) {
 		logging.level = 'silent';
 	}
-	const telemetry = new AstroTelemetry({ version: process.env.PACKAGE_VERSION ?? '' });
-
-	if (cmd === 'telemetry') {
-		try {
-			const subcommand = flags._[3]?.toString();
-			return await telemetryHandler.update(subcommand, { flags, telemetry });
-		} catch (err) {
-			return throwAndExit(err);
-		}
-	}
+	const telemetry = new AstroTelemetry({ version: ASTRO_VERSION });
 
+	// Special CLI Commands: "add", "docs", "telemetry"
+	// These commands run before the user's config is parsed, and may have other special
+	// conditions that should be handled here, before the others.
+	//
 	switch (cmd) {
 		case 'add': {
 			try {
+				telemetry.record(event.eventCliSession({ cliCommand: cmd }));
 				const packages = flags._.slice(3) as string[];
-				telemetry.record(
-					event.eventCliSession({
-						astroVersion: process.env.PACKAGE_VERSION ?? '',
```

Comment:
this isn't needed, since `astroVersion` is passed to the `AstroTelemetry` constructor above.